core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/SelectClause.kt

```diff
@@ -50,3 +52,6 @@ class SelectDistinctClause(private val expression: Expression, private vararg va
         )
     }
 }
+
+private fun getExpressionDopeQuery(expression: Expression) =
```

Comment:
that's looks a bit unclean. You change the generation of the queryString of an AliasedBucket inside the SelectClause. It should be done inside the toDopeQuery-Method of the AliasedBucket.

Reply:
How could I do that?

Reply:
So the difference is that once it is an _ExpressionBucket_ and once it is a _FromableBucket_ right? So we could split these into separate interfaces/classes for example

Reply:
Or another approach is making the AliasedBucket not inherit from Expression and add an additional AliasedExpressionBucket

Reply:
I added a ReferencedAliasedBucket